RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+17.7
+-----
+* [**] Self hosted sites that do not use Jetpack can now manage (install, modify active, uninstall) their plugins [#16675]
```

Comment:
Can you please add `(Don't apply to Jetpack app)` before the change description?

Reply:
Done! 426ad9ce0f16d464a39ef25756e601a540946b14